### PR TITLE
fix: remove unnecessary error throw

### PR DESCRIPTION
**Pull request by myarichuk**
- [x] Ready for review
- [ ] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Remove unnecessary error throw since the business logic dictates a NOP if a dependency object is invalid

### CODE REVIEW

File: test/parseNuspec-no-dependencies.spec.ts [hunk 1]
```diff
@@ -114,9 +114,4 @@ describe('parseNuSpec', () => {
       version: '472'},'dependencyName')).rejects.toThrow();
   });
 
-  it('should throw an error when the nuspec contains non-array dependencies tag', async () => {
```
Comment (jan-stehlik) [inline test/parseNuspec-no-dependencies.spec.ts:117]: can we delete this fixture now too? is it used anywhere else?

**Review by jan-stehlik — APPROVED** [review]
thanks @myarichuk :tada: , i have just a minor comment

File: test/parseNuspec-no-dependencies.spec.ts (hunk 1 above)
Comment (myarichuk) [reply test/parseNuspec-no-dependencies.spec.ts:117]: you are right! I will delete it

**Review by tonidevine1 — APPROVED** [review]
Looks good Michael!

**Pull request merged**